docs/tools/cli/project.rst

```diff
@@ -276,3 +276,30 @@ Removes a user with the supplied email address from the project.
 .. code:: shell
 
   avn project user-remove --project my-project alice@example.com
+
+
+Request project SBOM
+--------------------
+
+SBOM reports are generated per project and can be downloaded as long as the necessary permissions are set for the project. You can get the SBOM report download link for a project using the following command:
+
+``avn_project_generate_sbom``
```

Comment:
```suggestion
``avn_project_generate_sbom``
'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

```

Reply:
Hey, I'm sorry, I didn't realize this format was used throughout the article. My bad! I've added the heading line back in.

Reply:
no worries, it is added now :+1: